src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,43 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contact detail in the address book.
+ */
+public class Contact {
+
+    public String value;
```

Comment:
It's a good practice to limit all class members permission whenever possible.
Consider setting permission to private